### ISSUE TITLE
Search finds personal space

### ISSUE THREAD

**Issue by JanAckermann** [owncloud/ocis#7993]
The report search endpoint returns the users own personal space eg “admin”. 
This might be implemented as designed but showing this result in the Web search confuses the user and has no use case. 

Unfortunately filter this from the search result in Web is not enough as we still get the number of the found results in the response headers. 

I appreciate if we can omit this from the search report

**Comment by kulmann** [comment]
I agree - for me it's confusing to have the personal space root itself among the search results.

**Comment by kulmann** [comment]
@micbar and @tbsbdr what's you take on this? If you agree with the expected behaviour, what's the prio? I think it's a prio 3 bug...

**Comment by tbsbdr** [comment]
I'd say it depends on the effort. If <= 1h i'd fix it soonish. If more i'd track it as known issue, p4

**Comment by dragonchaser** [comment]
@kulmann after talking to @ScharfViktor and @kobergj the best approach would be to filter that out in web, since you have to iterate over that in web anyway to display it. Filtering that in the backend would be a waste of resources since we have to iterate over the whole resultset, which in 99% of all cases does not contain a personal space.

**Comment by AlexAndBear** [comment]
Understood, transferring to web and sort that out by myself
